### ISSUE TITLE
Remove twisted.spread.ui

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8329]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8329|
|Type|task|
|Created|2016-05-13 08:59:05Z|
|Branch|https://github.com/twisted/twisted/tree/nospreadui-8329|

```
[4:52pm] glyph:
hawkowl: yeah I think you might be in compatibility-exception territory if you want to fix that coverage problem
[4:52pm] hawkowl:
glyph: okay so. like
[4:52pm] hawkowl:
twisted.spread.ui, twisted.manhole, tkconch/ckeygen/etc
[4:52pm] hawkowl:
the former has zero tests and I am not sure how to confirm that it works
...
[4:54pm] glyph:
twisted.spread.ui... yeah it has to go, I think :-\
...
[4:55pm] glyph:
and like.. twisted.spread.ui depends on 'import gtk' and probably doesn't work at all as it is
```

<details><summary>Searchable metadata</summary>

```
trac-id__8329 8329
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__nospreadui_8329 nospreadui-8329
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1463129945839270 1463129945839270
changetime__1463837169079968 1463837169079968
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

Happy with this, please review. Builders are green.

Please see http://twistedmatrix.com/pipermail/twisted-python/2016-May/030325.html, this will need to wait a week (so minimum of the 21st to merge) + needs two more signoffs before a merge, but I see no reason why the review can't happen before then.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @hawkowl|
|-|-|

This branch depends on [#8330](https://github.com/twisted/twisted/issues/8330) as the twisted/manhole/ui/gtk2manhole.py contains a reference to twisted.spread.ui ... but should be good.

Looks good. Thanks!

Please merge ... after the wait :)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl set status to `closed`|
|-|-|

In changeset 89739f47976320104a7f3cf19e2a059e30d9811a
```
#!CommitTicketReference repository="" revision="89739f47976320104a7f3cf19e2a059e30d9811a"
Merge nospreadui-8329: Remove twisted.spread.ui

Author: hawkowl
Reviewer: adiroiban
Fixes: #8329
Compatibility Exception Signoffs: hawkowl, glyph, adiroiban
```
